src/inspection/InspectionUsers.tsx

```diff
@@ -62,7 +66,10 @@ const InspectionUsers: React.FC<PropTypes> = observer(({ inspection }) => {
               style={{ marginLeft: 'auto' }}
               buttonStyle={ButtonStyle.SECONDARY}
               size={ButtonSize.SMALL}
-              onClick={refetchRelations}>
+              onClick={(e: React.MouseEvent) => {
+                refetchRelations()
+                stopEventPropagation(e)
+              }}>
```

Comment:
I would rather create a new callback, `onRefetchRelations`, which stops the event propagation, than make a separate stopPropagation function.

```suggestion
onClick={onRefetchRelations}>
```